views/partials/forms/scan-email-form.hbs

```diff
@@ -15,7 +24,7 @@
     {{#if featuredBreach}}
       <input type="submit" value="{{ getString "find-out" }}"/>
     {{else}}
-      <input type="submit" value="{{ getString "check-for-breaches" }}"/>
+      <input class="add-metrics-flow-values" data-form_type="other" {{> analytics/fxa id="fx-monitor-check-for-breaches-blue-btn" }} type="submit" value="{{ getString "check-for-breaches" }}"/>
```

Comment:
I'm intentionally not using `.open-oauth`class as it needs to be conditional based if the user opts-in. `'.open-oauth` has other events attached to it that would fire regardless.